Makefile

```diff
@@ -50,4 +50,8 @@ lint:
 imports:
 	$(MAKE) --no-print-directory -C tm2      imports
 	$(MAKE) --no-print-directory -C gnovm    imports
-	$(MAKE) --no-print-directory -C gno.land imports
\ No newline at end of file
+	$(MAKE) --no-print-directory -C gno.land imports
+
+.PHONY: tidy
+tidy:
+	$(MAKE) --no-print-directory -C misc     tidy
```

Comment:
Can you please double-check? I vaguely recall creating something similar, possibly only in the CI. Just ensure that there isn't already an existing solution for the same purpose.

Reply:
It turns out we had not 1, but 2 tidy flows 🙂 

Fixed:
[9ab96d0](https://github.com/gnolang/gno/pull/1302/commits/9ab96d02565d0fdc682fa475cd2cbd3aeae682b2) and [a8fe8d3](https://github.com/gnolang/gno/pull/1302/commits/a8fe8d337369c31dbd69a372cd2c4382a6e94c15)